ui/src/pages/Questions/Detail/index.scss

```diff
@@ -16,15 +16,43 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+#reaction-button-tooltip {
+  .tooltip-inner {
+    padding: 0.2rem;
+    background-color: var(--bs-body-bg);
+    border: 1px solid var(--bs-border-color);
+    .btn {
+      background-color: var(--bs-body-bg);
+      border-color: transparent;
+      &:hover {
+        background-color: var(--bs-gray-100);
+      }
+      &:active {
+        background-color: var(--bs-gray-200);
+      }
+    }
+  }
 
-.answer-item {
-  border-top: 1px solid var(--an-answer-item-border-top);
+  .tooltip-arrow {
+    overflow: hidden;
+    bottom: calc(-0.4rem + 1px);
+    &::before {
+      border: 1px solid var(--bs-gray-500);
+      width: 50%;
+      height: 100%;
+      transform: translate(-50%, -50%) rotate(-45deg);
+      box-sizing: content-box;
+      left: 50%;
+      bottom: 50%;
+      background-color: white;
+    }
+  }
 }
 
```

Comment:
Through pseudo-elements, clicking on other areas can automatically close the tooltip of the emoji button.
```
.smile-btn.active::before{
  position: fixed;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  z-index: 80;
  display: block;
  cursor: default;
  content: " ";
}
```

Reply:
updated.